libs/portals/my-pages/health/src/screens/Questionnaires/AnsweredQuestionnaire.tsx

```
@@ -0,0 +1,191 @@
+    ':org',
+    org?.toLocaleLowerCase() ?? '',
+  ).replace(':id', id)
+
+  return (
+    <IntroWrapper
+      title={data?.getAnsweredQuestionnaire?.data[0]?.title || ''}
+      intro={formatMessage(messages.answeredQuestionnaireAnswered)}
+      loading={loading}
+      buttonGroupAlignment={'spaceBetween'}
+      buttonGroup={[
+        <Box>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)